notebooks/book2/14/linreg_height_weight_numpyro.ipynb

```diff
@@ -0,0 +1,939 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        plt.legend(fontsize = 7, loc = 'best', bbox_to_anchor = (0.3,-0.05),frameon = False)
</pre><p>make <code>frameon=True</code>,since legend are outside we need frame  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>